thentos-tests/src/Thentos/Test/Core.hs

```diff
@@ -184,7 +181,8 @@ createActionState dbname config = do
     connPool <- createDb dbname
     return $ ActionState (connPool, rng, config)
 
--- | Create a connection to an empty DB.
+-- | Create a connection to an empty DB.  (NOTE: There is no need for a 'destroyDb', since 'Pool'
```

Comment:
I'm generally not in favour of incorporating documentation from external libraries :). I'd also *imagine* `Pool` is sensibly-garbage collected, so it's not surprising behaviour.